### PR TITLE
Updated dockerfile structure

### PR DESCRIPTION
**Pull request by SergeyPoslavskiy**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by 0xxlegolas — APPROVED** [review]

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cp $KLAYTN_CONTRACT_DEPLOYMENT_DIR/networks-DexFactory.json /app/subgraphs/exchange/networks.json
```
Comment (odg0318) [inline docker-entrypoint.sh:3]: What do you think of adding this line?

```suggestion
#!/bin/bash

cp $KLAYTN_CONTRACT_DEPLOYMENT_DIR/networks-DexFactory.json /app/subgraphs/exchange/networks.json
```

File: example-Dockerfile [hunk 2]
```diff
@@ -1,28 +1,12 @@
 # better to stick with lts
```
Comment (odg0318) [inline example-Dockerfile]: Please rename this filename to Dockerfile.

File: docker-entrypoint.sh (hunk 1 above)
Comment (ghost) [reply docker-entrypoint.sh:3]: I have /bin/bash in entrypoint

File: example-Dockerfile (hunk 2 above)
Comment (ghost) [reply example-Dockerfile]: ok

File: docker-entrypoint.sh (hunk 1 above)
Comment (ghost) [reply docker-entrypoint.sh:3]: Added in script also

**Review by ghost — APPROVED** [review]

**Review by odg0318 — APPROVED** [review]

**Pull request merged**